spec/controllers/settings_controller_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'spec_helper'
 
 describe SettingsController, type: :controller do
+  render_views
 
+  let(:user) { create(:user) }
+
+  before do
+    allow(self.controller).to receive(:current_user).and_return(user)
```

Comment:
I think I had this on another functional test, but we have 

https://github.com/benwoody/shrtnr/blob/master/spec/support/authentication.rb#L3 

to help us out!